tcmur_cmd_handler.c

```diff
@@ -2308,6 +2319,320 @@ static int handle_inquiry(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return ret;
 }
 
+static uint8_t *get_cdb_usage_data(uint8_t op, uint8_t sa)
+{
+	static uint8_t usage[16];
+	uint8_t *buf = usage;
+
```

Comment:
Hey,

I think the patch looks ok.

Just FYI, it is taking me a while to check the cbd_usage bits. It is a not so fun task :)

I think some commands the kernel handles for us like report luns and the PGR ones are not handled here.

Reply:
@mikechristie I apologized that It was not until now that I read the mail. 
Thanks a lot.